### ISSUE TITLE
Alternatives for `flyspell-goto-next-error` and `flyspell-auto-correct-word`

### ISSUE THREAD

**Issue by meain** [astoff/jit-spell#5]
Thanks for the package. My personal workflow of flyspell involves the following two functions:

- `flyspell-goto-next-error` puts cursor on the next flyspell error
- `flyspell-auto-correct-word` chooses the first option, iterating between them on follow up invocations

Just wondering if you have considered them.

**Comment by astoff** [comment]
Did you try `jit-spell-correct-word`? I'm not in principle against adding a cycling command as in Flyspell, but I'm not convinced they are more usable.

**Comment by meain** [comment]
> Did you try jit-spell-correct-word?

Yup. `flyspell-auto-correct-word` fixes it to what I need the very first time more than 95% of the time avoiding having to select from a list. I can probably hack together something that just chooses the first selection automatically and that would mostly cover my usecase for correcting a word.

**Comment by astoff** [comment]
Okay, I fixed the sorting in `jit-spell-correct-word`, so now the top of the list is the right choice 95% of the time :-). You still need to do `C-; RET` instead of `C-;`, but there are advantages (you can accept the word, cycle with `C-;`, etc). Let me know what you think.

**Comment by meain** [comment]
That improves the situation quite a bit. It would be great to have something like `flyspell-goto-next-error` as well and ideally a `goto-previous` (flyspell does not have it currently). While I might rely on this to fix errors, I'll keep this in my toolkit to see if I have misspellings in my buffer (but with flyspell fixing the actual errors).
